pkg/helm/install.go

```diff
@@ -20,14 +20,14 @@ type InstallStep struct {
 type InstallArguments struct {
 	Step `yaml:",inline"`
 
-	Namespace string            `yaml:"namespace"`
-	Name      string            `yaml:"name"`
-	Chart     string            `yaml:"chart"`
-	Version   string            `yaml:"version"`
-	Replace   bool              `yaml:"replace"`
-	Set       map[string]string `yaml:"set"`
-	Values    []string          `yaml:"values"`
-	Wait      bool              `yaml:"wait"`
+	Namespace string                 `yaml:"namespace"`
+	Name      string                 `yaml:"name"`
+	Chart     string                 `yaml:"chart"`
+	Version   string                 `yaml:"version"`
+	Replace   bool                   `yaml:"replace"`
+	Set       map[string]interface{} `yaml:"set"`
```

Comment:
Since you changed this, you may want to update 'pkg/helm/testdata/install-input.yaml` and `TestMixin_UnmarshalInstallStep` and verify the change as well.